controller/Lists.php

```diff
@@ -83,15 +88,64 @@ public function index()
         $this->setView('Lists/index.tpl');
     }
 
+    /**
+     * @param RemoteSourcedListService $remoteSourcedListService
+     *
+     * @throws ExtensionException
+     * @throws core_kernel_persistence_Exception
+     */
+    public function remote(RemoteSourcedListService $remoteSourcedListService): void
+    {
+        tao_helpers_Scriptloader::addCssFile(Template::css('lists.css', 'tao'));
+
+        $this->defaultData();
+
+        $remoteListFormFactory = new tao_actions_form_RemoteList();
+        $remoteListForm = $remoteListFormFactory->getForm();
+
+        if ($remoteListForm === null) {
+            throw new RuntimeException('Impossible to create remote sourced list form');
+        }
+
+        if ($remoteListForm->isSubmited()) {
+            if ($remoteListForm->isValid()) {
+                $values = $remoteListForm->getValues();
+
+                $newList = $remoteSourcedListService->createList(
+                    $values[tao_actions_form_RemoteList::FIELD_NAME],
+                    $values[tao_actions_form_RemoteList::FIELD_SOURCE_URL],
+                    $values[tao_actions_form_RemoteList::FIELD_ITEM_LABEL_PATH],
+                    $values[tao_actions_form_RemoteList::FIELD_ITEM_URI_PATH]
```

Comment:
Are you sure `$values` array contains all of these items?

Reply:
`$values`  doesn't depend on request parameters, so I'd say that I am sure